contracts/modules/token/TokenController.sol

```diff
@@ -121,13 +121,18 @@ contract TokenController is ITokenController, ITokenControllerErrors, LockHandle
     token.addToWhiteList(_member);
   }
 
-  /// @dev Removes an address from the whitelist in the token.
+  /// @notice Removes a member from all assessor groups and the whitelist.
+  /// @dev Requires the member's token balance to be zero before removal.
   /// @param _member The address to remove.
   function removeFromWhitelist(address _member) public override onlyInternal {
     require(token.balanceOf(_member) == 0, MemberBalanceNotZero());
     token.removeFromWhiteList(_member);
   }
 
+  /// @notice Switches membership from one address to another, transferring all tokens and whitelist status.
```

Comment:
switching from one address to another is essentially the "whitelist status" transfer. remove the latter.